rpcs3/rpcs3qt/user_account.cpp

```diff
@@ -8,10 +8,8 @@
 LOG_CHANNEL(gui_log, "GUI");
 
 UserAccount::UserAccount(const std::string& user_id)
+    : m_user_id(user_id) // Setting userId.
```

Comment:
the comment can be omitted. it's 100% clear what the code does. I don't know why it was there in the first place

Reply:
Fix it.
[https://github.com/RPCS3/rpcs3/pull/9346/commits/a6c664dbe64c3ca82db6e4f0d81c81dcac99a59c](url)